static/app/types/event.tsx

```diff
@@ -522,6 +522,7 @@ export type PerformanceDetectorData = {
   causeSpanIds: string[];
   offenderSpanIds: string[];
   parentSpanIds: string[];
+  type: number;
```

Comment:
As discussed offline, let's delete this since we should be signifying the issue type from the `IssueType` enum available on the group object instead 

Last time I checked, the backend should not be sending a type property here to begin with! 😬

Reply:
@0Calories btw looked into it and we just serialize the entire perfproblem which is why the type is attached. Dunno if "it should not", cause if so someone should probably fix that

Reply:
Thanks for the heads up!

Reply:
Hey @wmak sorry, looks like there was some mismatch between the BE and FE contract, the `type` property is good to stay on the perfproblem! I'll look into a PR in the future to improve the typing on the frontend here